pkg/generated/openapi/openapi_test.go

```diff
@@ -51,8 +53,16 @@ func TestOpenAPIRoundtrip(t *testing.T) {
 			delete(roundTripped.Extensions, common.ExtensionV2Schema)
 			delete(value.Schema.Extensions, common.ExtensionV2Schema)
 
-			if !reflect.DeepEqual(value.Schema, roundTripped) {
-				t.Errorf("unexpected diff (a=expected,b=roundtripped):\n%s", cmp.Diff(value.Schema, roundTripped))
+			opts := []cmp.Option{
+				cmpopts.EquateEmpty(),
+				// jsonreference.Ref contains unexported fields. Compare
+				// by string representation provides a consistent
```

Comment:
the switch to go cmp I mean.

Reply:
Something about this API change caused our generated OpenAPI to contain `jsonreference.Ref` entries.  jsonreference.Ref contains unexported fields and caused this test to fail.  I'm a bit surprised I was the first to hit this TBH, since the generator has had the ability generate output like this all along, but this fixes the equality check in a sane/targeted way.